libraries/wasm-jit/Source/Runtime/WAVMIntrinsics.cpp

```diff
@@ -152,18 +157,19 @@ namespace Runtime
 			actualSignature ? asString(actualSignature).c_str() : "nullptr",
 			ipDescription.c_str()
 			);
-		causeException(elementValue == nullptr ? Exception::Cause::undefinedTableElement : Exception::Cause::indirectCallSignatureMismatch);
+		causeIntrensicException(elementValue == nullptr ? Exception::Cause::undefinedTableElement : Exception::Cause::indirectCallSignatureMismatch);
 	}
```

Comment:
Don't we need to wrap all these methods try-catch to catch `bad_alloc` or other exceptions that might leak through?

Reply:
I didn't see any thing in there that could throw, did I miss something maybe? I would tend to agree that it's somewhat of a land mine to leave them naked in case something changes in the future. However, at this point, it's not like we would ever be able to blindly sync with upstream anyways.

Reply:
The `std::string` could throw `bad_alloc`.

Reply:
Okay, while we're probably going down the toilet anyways if that ever happens, I did try to protect the couple cases where that could occur